src/Utils/PluginInfo.as

```diff
@@ -14,6 +14,24 @@ class TagInfo
 	}
 }
 
+class Changelog
+{
+	int m_siteID;
+	int64 m_postTime;
+	Version m_version;
+    bool m_isSigned;
+    string m_changeMessage;
+
+	Changelog(const Json::Value &in js)
+	{
+		m_siteID = js["id"];
+		m_postTime = js["posttime"];
+		m_version = Version(js["version"]);
+		m_isSigned = js["signed"];
+		m_changeMessage = js["changes"];
+	}
+}
+
```

Comment:
This could probably be in a new file. Also a better name might be `PluginChangelog`.

Reply:
can do! I'll go ahead and extract TagInfo while I'm at it since it's a similar type of holding class